evm2wast.c

```diff
@@ -0,0 +1,320 @@
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <dirent.h>
+#include <math.h>
+#include <errno.h>
+#include <stdarg.h>
+#include "evm2wast.h"
+#include "gadgets.h"
+int digits(unsigned long x) { return (floor(log10(abs(x))) + 1); }
```

Comment:
Should be `static` if we need this helper. Also why place it above the global variables and not next to the functions?